averages.js

```diff
@@ -0,0 +1,30 @@
+
+var total = Array.prototype.total = function() {
+  var array = this;
+  array.reduce(function (a, b) {
+    return a + b;
+  }, 0)
+}
+
+var mean = Array.prototype.mean = function() {
+  var array = this;
+  length = array.length
+  sum = array.reduce(function (a, b) {
+    return a + b;
+  }, 0)
+  return sum / length
+}
+
+var median = Array.prototype.median = function() {
```

Comment:
we probably don't want to add these functions (total, mean, and median) to the array's prototype. generally, we don't want to add things to JavaScript's built-in objects. we just want to create stand-alone functions and use them accordingly.